v3/src/components/free-tile-component.tsx

```diff
@@ -50,26 +48,17 @@
       setResizingTileId(mtile.tileId)
```

Comment:
In V2 resizing components is not continuous but has a delta that aids the user in lining things up nicely. Is this the place that increment should be used?

Reply:
Yes -- do you happen to recall the v2 grid size?

Reply:
Never mind; I found it. The v2 grid size is 5px, specified in `view_utilities.js`.

Reply:
In the interest of not holding up this PR I created a new story for the gridding behavior.